ajax/libs/vocalizer/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "vocalizer",
+  "filename": "vocalizer.min.js",
+  "version": "1.0.0",
+  "description": "Vocalizer is a plugin that help people learn to say your name",
+  "homepage": "https://www.vocalizer.io",
+  "keywords": [
+    "vocalizer",
+    "audio",
+    "pronunciation",
+    "names"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/atifazam/vocalizer"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/atifazam/vocalizer.git",
+    "basePath": "",
```

Comment:
The files are under the `vendor` directory, should set `basePath` to `"vendor"`, by the way, I prefer `"dist"` instead of `"vendor"` if it's possible, so that we can use pattern `"**/*"` in the file list to grab all the files under the folder automatically no matter how the structure changed.